inductiva/utils/templates.py

```diff
@@ -1,83 +1,74 @@
 """Utils related to template files."""
 
 import os
-import io
+import pathlib
 import glob
-from typing import Dict, List, Union
-from pathlib import Path
+import shutil
 
-from jinja2 import Environment, FileSystemLoader
+import jinja2
 
 from inductiva.utils.files import find_path_to_package
 
 TEMPLATES_PATH = find_path_to_package("templates")
 
 
-def replace_params(
-    template_path: str,
-    params: Dict,
-    output_file: Union[str, io.StringIO],
-    remove_template: bool = False,
-) -> None:
-    """Replaces parameters in a template file."""
+def render_from_string(template_string, **render_args):
+    """Render commands with the given arguments."""
 
-    template_path = Path(template_path)
+    template_env = jinja2.Environment(loader=jinja2.BaseLoader())
+    template = template_env.from_string(template_string)
+    string = template.render(**render_args)
 
-    template_dir = template_path.parent
-    template_filename = template_path.name
+    return string
 
-    environment = Environment(loader=FileSystemLoader(template_dir))
-    template = environment.get_template(template_filename)
-    stream = template.stream(**params)
-    stream.dump(output_file)
+
+def render_file(source_file, target_file, remove_template=False, **render_args):
```

Comment:
why should we have the `remove_template` argument? I thin it doesn't make sense to remove templates

Reply:
The remove template is because in the `add_dir` we copy the files first and then iterate over them to template (to also pass all the other files that aren't jinja). Hence, at the end we remove the .jinja template files.